documentation/en/default-lotus-miner-config.toml

```diff
@@ -257,6 +257,27 @@
       #Path = ""
 
 
+[IndexProvider]
```

Comment:
should there be a config option example here for how to disable the IndexProvider?

Reply:
This file is auto-generated from `node/config/types.go`, and will contain whatever defaults are set in the default config

Reply:
Added `Enable` flag to this config section, set to false by default until this PR is merged to master.